utils/styleguide/styleguide.base.config.js

```diff
@@ -156,6 +168,30 @@ const defaultStyleguideConfig = {
           loader: 'babel-loader',
           options: babelOptions
         },
+        {
+          test: /\.tsx?$/u,
+          loaders: [
+            {
+              loader: 'eslint-loader'
```

Comment:
Why the separation here for tsx into two different loaders?

Reply:
This loader uses `enforce: 'pre'` which I believe has to be enforced at a module rule level. We don't want to enforce that on `ts-loader` and `babel-loader`